src/pages/StyleGuideList.js

```diff
@@ -1,11 +1,19 @@
 /* @flow */
 
-import React from 'react';
-import Router from 'react-router';
-import components from '../components';
+import React from "react";
+import Router from "react-router";
+import components from "../components";
 
 export default class StyleGuideList extends React.Component {
-    li(componentName: string): ReactElement {
+    render(): ReactElement {
```

Comment:
Oh right. I thought there was an actual li somewhere because it's in a ul but now I see there never was, at least in this file.